src/Tools/dotnet-user-jwts/src/Helpers/JwtAuthenticationSchemeSettings.cs

```diff
@@ -70,6 +80,11 @@ public static void RemoveScheme(string filePath, string name)
             authentication[SchemesKey] is JsonObject schemes)
         {
             schemes.Remove(name);
+            if (authentication[DefaultSchemeKey] is JsonValue defaultScheme
```

Comment:
No tests for removing default scheme

Reply:
Good call. Updated the existing clear/remove tests to check for unsetting the DefaultScheme, also added a test to validate the DefaultScheme is not unset if the associated token isn't removed.